scripts/telemctl

```diff
@@ -18,6 +18,17 @@ declare -a SERVICES=(
 SCRIPT="$0"
 TELEM_DIR=/etc/telemetrics
 OPT_OUT_FILE=${TELEM_DIR}/opt-out
+TELEM_WRK_DIRS_CONF=/usr/lib/tmpfiles.d/telemetrics-dirs.conf
+
+create_work_dirs() {
+  # Creates dirs if missing, adjust ownership if exists
+  systemd-tmpfiles --create telemetrics-dirs.conf
```

Comment:
Even though passing the conf's basename here is valid, I would prefer to reuse the `TELEM_WRK_DIRS_CONF` variable again.